src/frontend/src/components/Posts/LoadMoreButton.jsx

```diff
@@ -23,12 +23,37 @@ const useStyles = makeStyles((theme) => ({
 
 function LoadMoreButton({ onClick }) {
   const classes = useStyles();
+  const $buttonRef = React.useRef(null);
+
+  // This will make the automatic infinite scrolling feature
```

Comment:
This is the same code again.  Why duplicate it?

Reply:
Sorry it was my mistake....